channels/mx.m3u

```diff
@@ -129,10 +117,6 @@ http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u8
 http://s5.mexside.net:1935/telemax/telemax/.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/0gL3FIN.png" group-title="",Teleritmo
 http://mdstrm.com/live-stream-playlist_400/57b4dc126338448314449d0c.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/qf0jh1I.png" group-title="",Telesecundaria
-https://srvstream01.televisioneducativa.gob.mx/hls/telesecundaria.m3u8
```

Comment:
Link works (Tested in VLC)

Reply:
Still not working. VLC output:
<img width="738" alt="Screenshot 2019-10-12 at 20 19 14" src="https://user-images.githubusercontent.com/7253922/66705319-97f15e80-ed2d-11e9-9c70-d3c1fb091964.png">

Reply:
Still works for me